vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -126,7 +126,7 @@ void ImageViewManager::UpdateProperties(
     const folly::dynamic &propertyValue{pair.second};
 
     if (propertyName == "source") {
-      setSource(canvas, propertyValue);
+      setSource(canvas, propertyValue, getMaxSize(reactDiffMap));
```

Comment:
There's no guarantee that the diffmap will contain a value for width or maxWidth. If only the source changes, then updateProperties can be called with just an entry for source in the diffmap and nothing else, and this will break.

Reply:
Added m_maxSize to ImageShadowNode though it now occurs to me that it might make more sense to track that in ReactImage...